src/bomi/player/mainwindow_m.cpp

```diff
@@ -269,6 +269,19 @@ auto MainWindow::Data::connectMenus() -> void
     connect(play(u"chapter"_q)[u"next"_q], &QAction::triggered, p,
             [seekChapter] () { seekChapter(+1); });
 
+    connect(play[u"state"_q], &QAction::triggered, p, [=] () {
+        QString str;
+        str.append(_MSecToString(e.time()));
+        str.append(u"/"_q);
+        str.append(_MSecToString(e.end()));
+        str.append(u" ("_q);
+        str.append(QString::number(e.time() * 100L / e.end()));
```

Comment:
If you really care about overflow, just cast type to double instead of long type integer. Also isn't 1%unit too inaccurate? I think 0.1% unit would be better.